pytrainer/gui/windowprofile.py

```diff
@@ -305,12 +305,15 @@ def on_addsport_clicked(self,widget):
         self.hidesportsteps()
         self.buttonbox.set_sensitive(0)
         self.addsport.show()
+        colorPixBuf = gtk.gdk.Pixbuf(gtk.gdk.COLORSPACE_RGB, False, 8, 250, 20)
+        colorPixBuf.fill(self.stored_color.rgba_val)
+        self.newcolor.set_from_pixbuf(colorPixBuf)
 
```

Comment:
Can you split the color fix into a separate PR against the Gtk3 branch (PR #120)? This would mildly conflict with that one, and I'd like to merge the main part (sport creation bug fix) for the next release and leave the gui changes for later.